Add updated benefits guidelines (#682)
2i2c-org/team-compass: people/compensation.md

## Compensation philosophy

:::{admonition} Aspirational
We are a young organization, and this page describes the salary policy that we aim to enact in the coming years.
It may not accurately reflect our current practices, but we share it for transparency and to provide a goal for our team to shoot for.
:::

:::{admonition}

> We are working on an update of our existing compensation policy. More details about the upcoming changes will be updated here as soon we we reached consensus and approval for the new policy.

Reviewer: ```suggestion
We are working on an update of our existing compensation policy. More details about the upcoming changes will be updated here as soon we reach consensus and approval for the new policy.
```

## Benefits

This section describes our benefits policy across all team members at 2i2c. 2i2c team members include people who are designated as employees and as contractors, depending on their role at 2i2c.

> Benefits are available to employees only and are managed through {term}`Code for Science and Society`’s relationship with a service partner or PEO. For example, employees who are US based and employed through {term}`Code for Science and Society`’s relationship with [Trinet](https://www.trinet.com/), will have access to health coverage and retirement benefits through Trinet. Employees based outside the US and employed through a relationship with a PEO like Remote will have access to benefits available in their region through that PEO. More details can be found at the [the CS&S employee handbook](https://drive.google.com/file/d/1anHo8P09gjGLnUfGj2ceSDxvJTYwMeS1/view?usp=sharing).

Reviewer: What about contractors who have no relationship with a PEO?

Author: Replied here: https://github.com/2i2c-org/meta/issues/444#issuecomment-1466964255

> Contractors are independent, not employed by CS&S, and are brought on to support 2i2c with their specific expertise. To support these contract relationships, 2i2c will cover some types of expenses related to the operation of independent contract business, such as would typically be changed by overhead at a larger organization. These expenses include healthcare, accounting services, and reimbursable equipment expenses. We estimate a 20% on top of the contractor monthly payment to cover for these sort of expenses.

Reviewer: ```suggestion
Contractors are independent, not employed by CS&S, and are brought on to support 2i2c with their specific expertise. To support these contract relationships, 2i2c will cover some types of expenses related to the operation of independent contract business, such as would typically be charged as overhead at a larger organization. These expenses include healthcare, accounting services, and reimbursable equipment expenses. We estimate a 20% on top of the contractor monthly payment to cover for these sort of expenses.
```

I think? Either way, this last phrase didn't parse correctly to me

### Time off

Our time off policy is documented in our [Taking time off](./time-off.md) section.

### Mechanism for payment

For anything that involves purchasing equipment, services, tickets, etc, we have these options:

> * Employee team members can pay via their own methods, and request a reimbursement via Ramp.

Reviewer 2: Link to https://compass.2i2c.org/en/latest/administration/reimburse.html as source of truth as for reimbursements for employees it's via invoice not ramp I believe.

Author: Updated! Thanks for the reminder!

* Contractor team members can pay via their own methods, and include a reimbursement line item in a monthly invoice.

Any team member can request that 2i2c pay for something with the 2i2c organization card.

### Equipment

We inherit the Equipment Policy for Code for Science and Society.
This defines a budget of $5,000 every 3 years for equipment for each team member.
This includes computer, desk, monitors, microphones, etc.
The goal is to make sure the team has the technical equipment they need to operate as a distributed team member that largely operates via video chat and text.

### Internet

We reimburse up to $100 a month for a high-speed internet subscription.

### Travel

2i2c will reimburse any work-related travel expenses for its team members.
Team members should get approval for travel (and rough budget) before any booking.

### Personal development

2i2c team members should identify opportunities to learn and improve the skills that are related to their role and their learning goals.
2i2c will cover the cost of these opportunities.

> We particularly encourage area leads and managers to identify these opportunities for others in their area (and for themselves)

Reviewer: ```suggestion
We particularly encourage area leads and managers to identify these opportunities for others in their area (and for themselves).
```

Missing punctuation

## Benefits

This section describes our benefits policy across all team members at 2i2c. 2i2c team members include people who are designated as employees and as contractors, depending on their role at 2i2c.

> Benefits are available to employees only and are managed through {term}`Code for Science and Society`’s relationship with a service partner or PEO. For example, employees who are US based and employed through {term}`Code for Science and Society`’s relationship with [Trinet](https://www.trinet.com/), will have access to health coverage and retirement benefits through Trinet. Employees based outside the US and employed through a relationship with a PEO like Remote will have access to benefits available in their region through that PEO. More details can be found at the [the CS&S employee handbook](https://drive.google.com/file/d/1anHo8P09gjGLnUfGj2ceSDxvJTYwMeS1/view?usp=sharing).

Reviewer 3: Can we group these benefits into a category of some sort? maybe payroll? (no idea if this is correct)

The reasoning is that we have this big section called _**"Benefits"**_ that describes the overall policy for employees & contractors. 

But then, there's the following sentence _"Benefits are available to employees only"_ which makes it a bit unclear that the extras defined in their own section below are available to everyone and not to employees only.

If we'd find a category for this, let's call it _payroll_ (again, I'm not sure this is correct), something like _"Payroll benefits are available to employees only"_ would be more clear that it refers to its own category of benefits.

What do you think?

Author: I restructure things a little bit more in my [latest commit](https://github.com/2i2c-org/team-compass/pull/682/commits/1cf40ee0d209e80db3e7c416e93a21b9da85ead9), let me know if that makes more sense to you!

Author: Resolving for now, but feel free to unresolve if you disagree.

> Contractors are independent, not employed by CS&S, and are brought on to support 2i2c with their specific expertise. To support these contract relationships, 2i2c will cover some types of expenses related to the operation of independent contract business, such as would typically be charged as overhead at a larger organization. These expenses include healthcare, accounting services, and reimbursable equipment expenses. We estimate a 20% on top of the contractor monthly payment to cover for these sort of expenses.

Reviewer 3: Hmm, I think this this should be a bit rephrased to match the info in https://github.com/2i2c-org/meta/issues/444#issuecomment-1466964255. After reading it, it seemed like _"expenses include healthcare, accounting services, and reimbursable equipment expenses"_ are reimbursed up to 20%.

Maybe something like the following:
```suggestion
Contractors are independent and not employed by CS&S or a PEO. They are brought on to support 2i2c with their specific expertise. They are not employees, so employees benefits are not available in this situation. 2i2c estimates that the employees specific benefits account for 20% of a monthly payment. In order to support role equity, this amount will be treated as overhead and will be added on top of the contractor monthly payment.
```

Author: I am not accepting the proposed changes, please check my comment on the parent issue for more details: https://github.com/2i2c-org/meta/issues/444#issuecomment-1469031926.
